<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号槽的5种连接方式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnMax">
      <property name="text">
       <string>最大化显示（!. SIGNAL/SLOT）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnNormal">
      <property name="text">
       <string>正常显示（2. 函数地址）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMin">
      <property name="text">
       <string>最小化显示（3. 设计师界面 - 转到槽）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>关闭窗口（4. 设计师界面 - 信号槽编辑器）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUpdate">
      <property name="text">
       <string>修改窗口标题（5. lambda表达式）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
